### ISSUE TITLE
Provide a REST implementation for remotemanager app operations

### ISSUE THREAD

**Issue by itziakos** [simphony/simphony-remote#7]
At the moment we just do a POST to the same form. This is suboptimal. I would prefer to have a REST api to start and stop containers, and then the webpage performs a request to this API and spawns the container. This would allow us to perform some nifty command line tool as well.

**Comment by stefanoborini** [comment]
https://github.com/tgonzales/mingus ?

**Comment by stefanoborini** [comment]
@itziakos currently available in deeply experimental form in pull request #80 and pull request #90 
Pull request #80 implements the basic concept. I then put it to the test in pull request #90 (branch on top of #80) where I found some shortcomings, so I changed the design to accommodate them.

Currently, PR #90 implements a CLI client that allows to list current applications, start and stop containers, and list currently active containers. It's not pretty and will need a lot more work of cleanup, refactoring of common code (with the HomeHandler), and use of the REST api via javascript, so at this point I don't really know where to stop with this PR.

What I can say is that, it its state, it satisfies the requirements, it's REST compliant, and we can build on top of it.
